### PR TITLE
Add types for Eris-Sharder

### PR DESCRIPTION
**Pull request by neowsl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/eris-sharder/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
+// Type definitions for Eris-Sharder 1.10
+// Project: https://github.com/discordware/eris-sharder
+// Definitions by: MathleteDev <https://github.com/mathletedev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Eris from 'eris';
+import { EventEmitter } from 'events';
+
+export class Cluster {
```
Comment (peterblazejewicz) [inline types/eris-sharder/index.d.ts:9]: Where does this entity comes from? It's not listed in the exports IMO:
https://github.com/discordware/eris-sharder/blob/dev/src/index.js#L4-L5

Comment (mathletedev) [reply types/eris-sharder/index.d.ts:9]: Oh sorry, I forgot to remove that export. I'll fix that.

Comment (peterblazejewicz) [reply types/eris-sharder/index.d.ts]: @mathletedev can you re-evaluate the outcome? 
TL;DR;
it's ok to `export interface Cluster` I think

I've re-read the source code, and I believe, your intent was correct. in the 'sendTo' client is expected to send cluster param, so at least the shape of the cluser should be exported. But instead of exporting class (which is not an option in native JS), one can export type (Interface), so consuming code could create an object represeting cluster. Sorry, I'm not an user of the package, just read the source code, thx!

Comment (mathletedev) [reply types/eris-sharder/index.d.ts]: Could you take a look at it? I changed the class to an interface and exported it. (Sorry, I'm still new to this and this is the first pull request I've done)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@mathletedev thx!

**Comment by mathletedev** [comment]
Ready to merge
